build.gradle

```diff
@@ -23,3 +23,13 @@ dependencies {
 test {
 	useJUnitPlatform()
 }
+
+plugins {
+  id "org.sonarqube" version "3.3"
+}
+
+sonarqube {
+  properties {
+    property "sonar.projectKey", "03495279ef60f968b4ff1ad4fb28e2e4e3fa7eca"
```

Comment:
This certainly shouldn't be publicly exposed, I suggest you revoke this API key from your side